app/views/dogs/index.html.erb

```diff
@@ -71,9 +72,9 @@
     search.addWidget(
       instantsearch.widgets.searchBox({
         container: '#q',
-        placeholder: 'Please search for name or breed',
+        placeholder: 'Search for name or breed',
         autofocus: false,
-        poweredBy: true
+        poweredBy: false
```

Comment:
You have to keep the "Powered by" because we use the free version of Algolia, see: https://www.algolia.com/pricing